app/config/galoy-instances.ts

```diff
@@ -6,9 +6,11 @@ const scriptHostname = (): string => {
   return scriptHostname
 }
 
-export type GaloyInstanceNames = "BBW" | "Staging" | "Local" | "Custom"
+export const possibleGaloyInstanceNames = ["BBW", "Staging", "Local", "Custom"]
```

Comment:
```suggestion
export const possibleGaloyInstanceNames = ["BBW", "Staging", "Local", "Custom"] as const
```

Reply:
type won't be narrowed as properly if you don't add the `as const`.  it would be a string instead of the individual values